packages/story-editor/src/utils/generateGroupName.ts

```diff
@@ -19,28 +19,39 @@
  */
 import { __, sprintf } from '@googleforcreators/i18n';
 
-export function getNextGroupNumber(groups) {
+interface GroupsProps {
+  [key: string]: {
+    name?: string;
+  };
+}
+
+export function getNextGroupNumber(groups: GroupsProps) {
   const nums = [0];
   const defaultName = __('Group', 'web-stories');
   for (const prop in groups) {
     if (!Object.prototype.hasOwnProperty.call(groups[prop], 'name')) {
       continue;
     }
-    const parts = groups[prop]?.name.split(' ');
+    const { name = '' } = groups[prop];
+    if (!name) {
+      continue;
+    }
```

Comment:
Maybe just
`if (!groups[prop]?.name) return;`?

Reply:
I can try it, but I tried a number of these and got a lint error everytime.

Reply:
Removed